apps/omg_watcher_rpc/priv/swagger/info_api_specs/configuration/schemas.yaml

```diff
@@ -0,0 +1,15 @@
+ConfigurationSchema:
+  type: object
+  properties:
+    deposit_finality_margin:
+      type: integer
+      format: int256
+    contract_semver:
+      type: string
+      format: binary
```

Comment:
I just realized that `format: binary` is [for describing files](https://swagger.io/docs/specification/data-models/data-types/#format). I think we can remove the `format` attribute.

Looks like you already removed it from some files but not all. Also applies to `network` below, and a few more other files.

Reply:
I guess I can remove this from everywhere right?